stack/core/src/main/java/org/apache/usergrid/corepersistence/CpEntityManagerFactory.java

```diff
@@ -347,67 +363,23 @@ public void deleteApplication(UUID applicationId) throws Exception {
                 }
             });
         }
-        final ApplicationEntityIndex aei = entityIndexFactory.createApplicationEntityIndex(deletedAppScope);
-        final Observable deleteNodeGraph = managementGraphManager.deleteNode(deletedAppId, Long.MAX_VALUE);
+        final ApplicationEntityIndex aei = entityIndexFactory.createApplicationEntityIndex(applicationScope);
+        final GraphManager managementGraphManager = managerCache.getGraphManager(managementAppScope);
+        final Observable deleteNodeGraph = managementGraphManager.deleteNode(applicationId, Long.MAX_VALUE);
```

Comment:
Deleting the old app entity makes me nervous. In the event we encounter a bug, we wont' have any roll back strategy.  Can we just migrate to the new info, then possibly delete the old values in our next release after it's been deployed and we ensure it's solid?